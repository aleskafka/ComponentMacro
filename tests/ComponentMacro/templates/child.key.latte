
<Header n:component>
	<h1 n:child="title">{$title} <small>{$small}</small></h1>
</Header>
