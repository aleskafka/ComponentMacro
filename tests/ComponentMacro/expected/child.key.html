
<div class="header">
 	<h1>Hello <small>world</small></h1>

</div>
